src/parser.js

```diff
@@ -565,6 +577,8 @@ const parser = (() => {
         terminal("in"); //
         prefix("-"); // unary numeric negation
         infix("~>"); // function application
+        infix("?:"); // default value
+        infix("??"); // coalescing operator
```

Comment:
Slightly more involved, but I think we can do the same here, since this is syntactic sugar for `$exists(a) ? a : b`:
```
infix("??", operators['??'], function (left) {
    this.type = 'condition';
    this.condition = {
        type: 'function',
        value: '(',
        procedure: {type: 'variable', value: 'exists'},
        arguments: [left]
    };
    this.then = left;
    this.else = expression(0);
    return this;
});
```
Then there is nothing extra to do in the evaluator.